R/scale-viridis.r

```diff
@@ -123,15 +124,23 @@ scale_fill_viridis_c <- function(..., alpha = 1, begin = 0, end = 1,
 scale_colour_viridis_b <- function(..., alpha = 1, begin = 0, end = 1,
                                    direction = 1, option = "D", values = NULL,
                                    space = "Lab", na.value = "grey50",
-                                   guide = "coloursteps", aesthetics = "colour") {
-  binned_scale(
-    aesthetics,
-    "viridis_b",
-    gradient_n_pal(
+                                   guide = "coloursteps", aesthetics = "colour", pal_full_range = FALSE) {
+  if(pal_full_range) {
```

Comment:
Why does this need to be an option? Wouldn't you always want to use the full range?

Reply:
as per our discussion in #4372 with @clauswilke @yutannihilation: both ways of doing it could make sense

Reply:
Can you please provide an argument as to why the current behaviour would ever be desired?

Reply:
I can't say that I can. Maybe @clauswilke @yutannihilation?

Reply:
@thomasp85 should issue a final ruling, but I'm pretty sure the original behaviour was a mistake, and we should just always use the full range. We'll need to flag this as a visual change in the NEWS.

Reply:
Agree - we should just go with the approach in this PR. The original implementation was an error on my part